lib/legacy_mempool.go

```diff
@@ -839,17 +839,15 @@ func (mp *DeSoMempool) addTransaction(
 
 	// Add it to the universal view. We assume the txn was already added to the
 	// backup view.
-	_, _, _, _, err = mp.universalUtxoView._connectTransaction(mempoolTx.Tx, mempoolTx.Hash, int64(mempoolTx.TxSizeBytes), height,
-		false /*verifySignatures*/, false /*ignoreUtxos*/)
+	_, _, _, _, err = mp.universalUtxoView._connectTransaction(mempoolTx.Tx, mempoolTx.Hash, int64(mempoolTx.TxSizeBytes), height, 0, false, false)
```

Comment:
Here I suspect that using 0 as the value may cause issues versus, for example, using the current timestamp.

Reply:
Is there any downside to using the current timestamp here rather than zero? I didn't see any. I don't think we've determined how the cutover to the new mempool is going to happen yet, and using the current timestamp will likely make your testing easier.